nilearn/plotting/find_cuts.py

```diff
@@ -210,7 +210,15 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     data = _smooth_array(data, affine, fwhm='fast')
 
     if spacing == 'auto':
-        spacing = max(int(.5 / n_cuts * data.shape[axis]), 1)
+        if n_cuts > 0.:
+            spacing = max(int(.5 / n_cuts * data.shape[axis]), 1)
+        else:
+            message = ("If the input 'cut_coords' is an integer, it only "
+                       "represents the number of slices to display. It should "
+                       "be given with value more than 0. or If you want to "
+                       "display in a coordinates of each direction, it should "
+                       "be given as scalar represented as [%.0f]." % n_cuts)
```

Comment:
I find your error message very verbose. I think you should go straight to the point. Something like:
"The number of cuts in direction '?' must be an integer comprised between 0 and this_shape. You provided ...".